docs/articles/faq/README.md

```diff
@@ -259,4 +262,54 @@ and [t.resizeWindow](../documentation/test-api/actions/resize-window.md#setting-
 to control the browser window size. If the test passes, it means your webpage hides
 the target element when the window is resized to smaller dimensions.
 
-Finally, try updating TestCafe to the latest version to see if the problem persists.
\ No newline at end of file
+Finally, try updating TestCafe to the latest version to see if the problem persists.
+
+### TestCafe reports that a request has failed. What are the possible reasons?
+
+When TestCafe does not receive a successful response from a server, it outputs the following error:
+
+```text
+A request to https://www.example.com has failed. Use quarantine mode to perform additional attempts to execute this test.
+```
+
+[Quarantine mode](../documentation/using-testcafe/command-line-interface.md#-q---quarantine-mode) helps you complete the tests if the problem is occasional.
+
+However, we recommend that you determine the cause of this issue and address it.
+
+The following situations lead to this error most frequently:
+
+#### The Web server is not responding
+
+Check if the Web and DNS servers are online and configured to accept requests to this URL.
+
+#### Unstable or improperly configured network connection
+
+To fix network issues, try the following:
+
+* Check the network connection's settings.
+* Ensure that your network equipment works properly. If possible, try to establish direct connection to the Internet/Web server.
+* Check the proxy server's settings or try a different proxy server.
+* Use VPN.
+* Connect to a different network.
+
+#### Not enough resources in the container or CI system
+
+If you run TestCafe in a container or CI system, try the following steps to diagnose resource shortage:
+
+* Increase the container's resource limits.
+* Set the [concurrency factor](../documentation/using-testcafe/common-concepts/concurrent-test-execution.md) to `1`.
+* Deploy the application's Web server on a separate machine.
+* Run tests on a local device outside a container.
+
+If this fixes the tests, it indicates that they require additional resources. You can address this in the following ways:
+
+* Adjust the container's or environment's settings to allocate more resources.
+* If you use a commercial CI system, ask the service provider for an upgrade or consider a different CI service with better hardware or smaller loads.
+
+TestCafe users have positive experience with the following CI systems:
```

Comment:
"TestCafe users have positive experience with the following CI systems" and I'm not sure about the point of this sentence

Reply:
Why is it necessary to say "have positive experience"? and do you mean 'a' positive experience?